### ISSUE TITLE
CSS @supports Firefox 69

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/sprints#1962]
See https://trello.com/c/TMRyYXYS/223-css-supports-fx-69 for the original trello card

# Acceptance criteria

1. - [ ] [Enable @supports selector() by default.](https://bugzilla.mozilla.org/show_bug.cgi?id=1513643)

**Comment by rachelandrew** [comment]
PR for spec data as we are missing Conditional Level 4 https://github.com/mdn/kumascript/pull/1215

**Comment by rachelandrew** [comment]
This is pretty much ready, there have already been examples added to the docs and the BCD is up to date.

Once my PR above is merged I can correct the spec table.

I've added a line to the release notes.

@chrisdavidmills I think this would be a good candidate for a Hacks Blog Post. I have made an issue for writing a guide to using feature queries, so perhaps I could do that in the next sprint and also a post which could reference that and the support of selector?
